hyrax/config/environments/development.rb

```diff
@@ -54,6 +54,7 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+  config.web_console.whitelisted_ips = ["172.0.0.0/8", '192.168.0.0/16', '127.0.0.1']
```

Comment:
these are the docker / local dev addresses, causing the web_console to work even if you're using docker if (and only if) you ware also in RAILS_ENV=development mode.